docs/migration-guide.md

```diff
@@ -31,6 +31,55 @@ sigv4:
   region: us-east-1
 ```
 
+## Tempo: `push_config` deprecation
+
+NOTE: `push_config` is still being supported as it was before v0.14.0.
```

Comment:
I'm not very convinced about this paragraph. Maybe it should be dropped or placed elsewhere.

Reply:
I'd just change the wording here, something like:

> `push_config` is now deprecated in favor of a `remote_write` array which allows for sending spans to multiple endpoints. `push_config` will be removed in a future release, and it is recommended to migrate to `remote_write` as soon as possible.

WDYT? Feel free to play around with the wording.

Reply:
I like that much better. Thanks.